tests/unit/test_code_generation.py

```diff
@@ -322,6 +322,26 @@ def test():
 """
     )
 
+    before_code = """\
+@udff
```

Comment:
Typo? I can't tell if it is supposed to be `@udtf` or `@udf` lol

Reply:
it is intentional to test a user-provided decorator starting from `@udf` but it's not `@udf` 🤦

Reply:
LOL. Let's add a comment then 😂